projects/1.16/assets/brewevolution/brewevolution/lang/zh_cn.json

```
@@ -0,0 +1,77 @@
+  "block.artofbeer.hops_crop_stage_4": "啤酒花作物：阶段4",
+  "block.brewevolution.beer_sign": "啤酒标志",
+  "item.brewevolution.calvados_bottle_item": "瓶装卡尔瓦多斯",
+  "block.artofbeer.hops_crop_stage_5": "啤酒花作物：阶段5",
+  "block.artofbeer.hops_crop_stage_2": "啤酒花作物：阶段2",
+  "block.artofbeer.hops_crop_stage_3": "啤酒花作物：阶段3",
+  "item.brewevolution.full_mug_item": "大杯装啤酒",
+  "block.artofbeer.hops_crop_stage_0": "啤酒花作物：阶段0",
+  "block.artofbeer.hops_crop_stage_1": "啤酒花作物：阶段1",
+  "item.brewevolution.vodka_bottle_item": "瓶装伏特加",
+  "block.brewevolution.hops_crop_stage_0": "啤酒花作物：阶段0",
+  "item.artofbeer.hops_seeds": "啤酒花种子",
+  "block.brewevolution.rum_glass": "杯装朗姆酒",
+  "item.brewevolution.dried_bread": "面包干",
+  "item.brewevolution.mollasses": "糖蜜",
+  "item.brewevolution.mead_horn": "角杯装蜜糖酒",
+  "item.brewevolution.empty_bottle": "空瓶",
+  "item.brewevolution.liqueur_glass_item": "杯装甜浆果利口酒",
+  "block.brewevolution.beer_sign_on_wall": "啤酒标志",
+  "block.artofbeer.beer_sign_on_wall": "啤酒标志",
+  "block.brewevolution.alambic": "蒸馏器",
+  "item.brewevolution.cheese": "奶酪",
+  "block.brewevolution.cider_glass": "杯装苹果酒",
+  "item.brewevolution.apple_juice_bucket": "苹果汁桶",
+  "item.brewevolution.cider_glass_item": "杯装苹果酒",
+  "item.artofbeer.hops": "啤酒花",
+  "block.brewevolution.just_a_glass": "空玻璃杯",
+  "block.brewevolution.planche_apero_stage_1": "开胃菜：阶段1",
+  "block.brewevolution.planche_apero_stage_2": "开胃菜：阶段2",
+  "item.brewevolution.crushed_apple": "碎苹果",
+  "block.brewevolution.planche_apero_stage_3": "开胃菜：阶段3",
+  "item.brewevolution.full_dried_bread": "面包干",
+  "block.artofbeer.beer_barrel": "啤酒桶",
+  "block.brewevolution.vodka_bottle": "瓶装伏特加",
+  "block.brewevolution.press": "榨汁机",
+  "item.brewevolution.kvass_mug": "大杯装格瓦斯",
+  "block.brewevolution.sweet_berry_liqueur_bottle": "瓶装甜浆果利口酒",
+  "block.brewevolution.hops_crop_stage_3": "啤酒花作物：阶段3",
+  "item.brewevolution.hops_seeds": "啤酒花种子",
+  "block.brewevolution.hops_crop_stage_4": "啤酒花作物：阶段4",
+  "block.brewevolution.hops_crop_stage_1": "啤酒花作物：阶段1",
+  "subtitles.pressing": "榨汁机：粉碎",
+  "block.brewevolution.hops_crop_stage_2": "啤酒花作物：阶段2",
+  "block.artofbeer.beer_sign": "啤酒标志",
+  "block.brewevolution.hops_crop_stage_7": "啤酒花作物：阶段7",
+  "block.brewevolution.vodka_glass": "杯装伏特加",
+  "item.brewevolution.cider_bottle_item": "瓶装苹果酒",
+  "block.brewevolution.hops_crop_stage_5": "啤酒花作物：阶段5",
+  "block.brewevolution.hops_crop_stage_6": "啤酒花作物：阶段6",
+  "block.brewevolution.planche_apero_empty": "空的开胃菜板",
+  "block.brewevolution.empty_mug": "空大杯",
+  "item.brewevolution.rum_bottle_item": "瓶装朗姆酒",
+  "item.artofbeer.full_mug_item": "满大杯",
+  "block.brewevolution.cidre_bottle": "瓶装苹果酒",
+  "block.brewevolution.mead_horn_block": "角杯装蜜糖酒",
+  "item.brewevolution.hops": "啤酒花",
+  "block.brewevolution.kvass_mug_block": "大杯装格瓦斯",
+  "item.brewevolution.calvados_glass_item": "瓶装卡尔瓦多斯",
+  "block.brewevolution.beer_barrel": "酿酒桶",
+  "block.brewevolution.planche_apero": "开胃菜板"

```

```suggestion
  "block.brewevolution.planche_apero": "盘装开胃酒"
```
https://en.wikipedia.org/wiki/Ap%C3%A9ritif_and_digestif
> An apéritif is an alcoholic beverage usually served before a meal to stimulate the appetite, and is usually dry rather than sweet.

https://en.wikipedia.org/wiki/Platter_(dishware)
> A platter is a large type of dishware used for serving food.